generator/booking_quality_check/booking_quality_check.cpp

```diff
@@ -139,18 +139,19 @@ feature::GenerateInfo GetGenerateInfo()
 struct SampleItem
 {
   enum MatchStatus {Uninitialized, Yes, No};
+  using ObjectId = SponsoredDataset::ObjectId;
```

Comment:
Generally speaking it's not a good idea to have a single type for identifiers of sponsored object. I.e. ids of hotels from booking and, say, restaurants from opentable will have the same type, which is not good. So from my point of view it much better to have different types.